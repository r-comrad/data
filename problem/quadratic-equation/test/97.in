6425 8130 -7128
