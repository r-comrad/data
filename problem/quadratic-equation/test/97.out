two solutions
0.596016
-1.861386
